review-tools/ghmerge

```diff
@@ -59,9 +60,14 @@ set -- `python -c '
 from __future__ import print_function
 import json, sys;
 print(str(json.load(sys.stdin)["head"]["label"]).replace(":", " "))' </tmp/gh$$`
-rm /tmp/gh$$
 WHO=$1
 BRANCH=$2
+set -- `python -c '
+from __future__ import print_function
+import json, sys;
+print(str(json.load(sys.stdin)["head"]["repo"]["ssh_url"]))' </tmp/gh$$`
+REPO=$1
+rm /tmp/gh$$
```

Comment:
Why not make it a single python call which loads the json file once and prints all three arguments?

Reply:
I was thinking the same when I copied the Python code for simplicity.
I'm not a Python programmer so far, but meanwhile found how to do it nicely.